src/components/mainContent/Navbar.js

```diff
@@ -18,7 +20,19 @@ export default function NavBar(props) {
       <button className="button" onClick={dropdownHandler}>
         <div className={`triangle ${dropdownOpen ? "up" : ""}`}></div>
         {dropdownOpen && (
-          <div className="dropdown-placeholder">dropdown menu</div>
+          <div class="dropdown">
+            <div class="dropdown-content">
+              <BrowserRouter>
+                <Link to='/' >Landing Page</Link>
+                <Link to='/'>About Us</Link>
+                <Link to='/'>Case Study #1</Link>
+                <Link to='/'>Case Study #2</Link>
+                <Link to='/'>Case Study #3</Link>
+                <Link to='/'>Conclusion</Link>
+                <Link to='/'>Resources</Link>
+              </BrowserRouter>
+            </div>
+          </div>
```

Comment:
Hm, so this isn't exactly how `react-router` is supposed to be configured, but it's super confusing so don't worry about it!

Long story short, you need to do a few things:

* at the root of the entire app (e.g. at `index.js` or `App.js`), you need to wrap the entire app in a `<BrowserRouter>` or some other `<Router>` tag - this tells react router that everything **inside that tag** belongs to the router.
* next, you need to add a `<Switch>` component with individual `<Route>` paths - this tells React what to render, given the Route. You should def check the docs for more info!
* you can keep your `<Link>` tags where they are right now, that's good!
* I would recommend using a `<nav>` component for accessibility!

I'd definitely check out [the quick start guide](https://reactrouter.com/web/guides/quick-start) for more info!